assets/src/styles/main.scss

```diff
@@ -52,6 +52,9 @@ body {
     margin-right: auto;
     @include u-maxw('desktop');
   }
+  .beta-tag {
+    @include u-bg('accent-warm-dark');
```

Comment:
I was hoping for something like the old Marquee tag, with a rainbow of flashing rotating letters, but I guess this will suffice ;)

Reply:
That would be cool but also very 1990's!